xrpl/models/base_model.py

```diff
@@ -54,7 +72,18 @@ def _key_to_json(field: str) -> str:
         1. 'TransactionType' becomes 'transaction_type'
         2. 'value' remains 'value'
         3. 'URI' becomes 'uri'
+
+        This function accepts inputs in camelCase only
+
+    Raises:
+        XRPLModelException: If the input is invalid
     """
+    if is_snake_case(field):
+        raise XRPLModelException(
+            "_key_to_json accepts input strings in the camelCase format only, "
```

Comment:
```suggestion
            f"Key {field} is not in the proper XRPL format."
```
Errors should be understandable by someone who does not know the internals of the repo. Anyone who is using xrpl-py isn't going to be using `_key_to_json`, therefore the error shouldn't specify it.

Reply:
fixed in [7405ba3](https://github.com/XRPLF/xrpl-py/pull/694/commits/7405ba3e197f840519629679d2da418639d5e0d6)